docs/website/blog/2023-06-26-dlthub-gptquestion2.md

```diff
@@ -0,0 +1,103 @@
+---
+slug: trained-gpt-q&a-2
+title: "`dlt` AI Assistant provides answers you need!"
+image: /img/blog_gpt_QA2.jpg
+authors:
+  name: Tong Chen
+  title: Data Engineer Intern at dltHub
+  url: https://github.com/TongHere
+  image_url: https://iili.io/HP9uKIn.jpg
+tags: [trained gpt-4, Q&A, dlt ]
+
+--- 
+Hi there! 👋
+
+In this article, we tried to ask our `dlt` AI Assistant- the conversational companion for the dltHub docs. We asked some questions about how `dlt` helps with Data pipeline maintenance and advanced `dlt`  project processes. Let’s see how it gave us answers! 
+       
```

Comment:
A little more context here would be nice. Maybe a background of how maintenance is a problem and that dlt can help reduce it. And then go on to answering how using ChatGPT.